vncproxy.install

```diff
@@ -1,5 +1,10 @@
-etc: [
-  "setup.ml"
-  "setup.data"
-  "setup.log"
+lib: [
```

Comment:
This file is auto-generated by `jbuilder`, needs to be deleted

Reply:
Ooops, indeed I have put it in .gitignore! 😳